### ISSUE TITLE
Smithsonian partnered with The Carpentries.

### ISSUE THREAD

**Issue by maurolepore** [forestgeo/forum#13]
_From @maurolepore on March 9, 2018 18:54_

The Smithsonian Institution has recently partnered with The Carpentries. The Carpentries -- [Software Carpentry](https://software-carpentry.org/) and [Data Carpentry](http://www.datacarpentry.org/) -- help researchers worldwide to become more productive by providing free training on data and software management.

Two Data Carpentry workshops have been delived to date (March 2018), one at NMNH and one at SERC. At SERC, for example, the workshop covered [this content](https://smithsonianworkshops.github.io/2018-03-07-serc/). 

Two more workshops are expected during 2018. If you would like to participate, keep alert because once registration opens the available slots (~30-40) tend to fill extremely fast.

_Copied from original issue: forestgeo/learn#63_

**Comment by gonzalezeb** [comment]
When are you bringing this to SCBI? Many of us would like to participate!

**Comment by maurolepore** [comment]
The Carpentries allow us a total of four official workshops each year. Two have already been delivered and I don't think the other two are scheduled yet. But be sure I'll let you know if they confirm.

Having said that, there is no limit to how many unofficial workshops we run. It's totally legal to run identical workshops as long as we acknowledge that they are based on The Carpentries materials. But event better -- in my opinion -- we can adapt the content to target our specific needs.

In short, it's up to us. We can make it happen if there is interest.
